Automattic/jetpack
# Jetpack: iterate over VPBlock UI errors

This PR iterates over the error messages shown in the UI, trying to follow the core designs:

Fixes #

#### Changes proposed in this Pull Request:

*

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to block editor
* Add a VPBlock
* Generate an error

before | after
------|-----
<img width="662" alt="image" src="https://user-images.githubusercontent.com/77539/178481908-fee61ec5-5f88-4a01-ba05-8b66ba7a0ffa.png"> | <img width="662" alt="image" src="https://user-images.githubusercontent.com/77539/178481647-a0e83074-69f0-46db-96e2-9de5ffa1804b.png">




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202586735461853
